app/src/main/java/edu/gatech/spacebarz/buzzshelter/model/Shelter.java

```diff
@@ -1,115 +1,116 @@
+        this.uid = uid;
     }
-
-    public String getShelterName() {
-        return shelterName;
+    public String getName() {
+        return name;
     }
-
-    public void setShelterName(String shelterName) {
-        this.shelterName = shelterName;
+    public void setName(String name) {
+        this.name = name;
     }
-
-    public boolean isHasFamily() {
-        return hasFamily;
+    public String getCapacityStr() {
+        return capacityStr;
     }
-
-    public void setHasFamily(boolean hasFamily) {
-        this.hasFamily = hasFamily;
+    public void setCapacityStr(String capacityStr) {
+        this.capacityStr = capacityStr;
     }
-
-    public boolean isVet() {
-        return isVet;
+    public String getRestrictions() {
+        return restrictions;
     }
-
-    public void setVet(boolean vet) {
-        isVet = vet;
+    public void setRestrictions(String restrictions) {
+        this.restrictions = restrictions;
     }
-
-    public Gender getGender() {
-        return gender;
+    public String getAddress() {
+        return address;
     }
-
-    public void setGender(Gender gender) {
-        this.gender = gender;
+    public void setAddress(String address) {
+        this.address = address;
     }
-
-    public Age getAge() {
-        return age;
+    public String getNotes() {
+        return notes;
     }
-
-    public void setAge(Age age) {
-        this.age = age;
+    public void setNotes(String notes) {
+        this.notes = notes;
     }
-
-    public String getKey() {
-        return key;
+    public String getPhone() {
+        return phone;
     }
-
-    void setKey(String key) {
-        this.key = key;
+    public void setPhone(String phone) {
+        this.phone = phone;
     }
-
-    public String getPhoneNum() {
-        return phoneNum;
+    public int getCapacityNum() {
+        return capacityNum;
     }
-
-    public void setPhonNum(String phonNum) {
-        this.phoneNum = phonNum;
+    public void setCapacityNum(int capacityNum) {
+        this.capacityNum = capacityNum;
     }
-
-    public String getSpecialNotes() {
-        return specialNotes;
+    public double getLat() {
+        return lat;
     }
-
-    public void setSpecialNotes(String specialNotes) {
-        this.specialNotes = specialNotes;
+    public void setLat(double lat) {
+        this.lat = lat;
     }
-
-    public Location getAddress() {
-        return address;
+    public double getLon() {
+        return lon;
+    }
+    public void setLon(double lon) {
+        this.lon = lon;
+    }
+    public Gender getGender() {
+        return gender;
+    }
+    public void setGender(Gender gender) {
+        this.gender = gender;
+    }
+    public boolean getVeteran() {
+        return veteran;
+    }
+    public void setVeteran(boolean veteran) {
+        this.veteran = veteran;
     }
 
-    public void setAddress(Location address) {
-        this.address = address;
+
+    public enum Gender {
+        MALE, FEMALE, ALL;
```

Comment:
Rather than having an ALL value, it may be better to just use a Set<Gender> object to store the allowed genders (may become more applicable when we store restrictions)

Reply:
Set object works. Easy to change in CSV uploader if we decide to.

Reply:
Ok, I'll hold off until we change restrictions to an enum and then we can change both of them to enum sets.